src/components/SeoForm/SeoForm.tsx

```diff
@@ -89,6 +94,7 @@ const SeoForm: React.FC<SeoFormProps> = props => {
   const intl = useIntl();
   const [expanded, setExpansionStatus] = React.useState(false);
   const toggleExpansion = () => setExpansionStatus(!expanded);
+  const shouldDisplayHelperText = () => helperText && !expanded;
```

Comment:
It doesn't have to be a function 🤷‍♂️

```suggestion
  const shouldDisplayHelperText = helperText && !expanded;
```

Reply:
Tru 😅 Changed